docs/user-guide/features.rst

```diff
@@ -33,6 +33,38 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Modularity
+^^^^^^^^^^
+
+Pulp supports following modularity_ repository mirroring use cases:
+
+* synchronization of the modularity metadata content, the
+  ``repodata/*modules.yaml.gz`` file, with either the repository immediate or
+  on-demand synchronization
+
+* publication of the modularity metadata with the repository publication
+
+* copy of the modularity metadata with the repository copy
+
+* modules published thru Pulp are consumable by the ``dnf`` client
+
+More advanced use cases are planned, especially those following from the ability
+of Pulp processing the modularity metadata file.
```

Comment:
"More advanced use cases are planned, especially those dealing with the ability of Pulp to process modularity metadata files."